clientdb/sidecar.go

```diff
@@ -171,3 +285,64 @@ func readSidecar(sourceBucket *bbolt.Bucket, id []byte) (*sidecar.Ticket,
 
 	return sidecar.DeserializeTicket(bytes.NewReader(sidecarBytes))
 }
+
+func storeBidTemplate(bidBucket *bbolt.Bucket, bid *order.Bid, ticketNonce order.Nonce) error {
+	var w bytes.Buffer
+	if err := SerializeOrder(bid, &w); err != nil {
+		return err
+	}
+
+	err := storeOrderTX(bidBucket, ticketNonce, w.Bytes(), nil)
+	if err != nil {
+		return err
+	}
+	err = storeOrderMinUnitsMatchTX(
```

Comment:
lolol yeah I had that same itch...if we ended up doing that, then IMO it would def make sense to just store the fully serialized order in the ticket, as that at that point there'd be no key/bucket structure we'd want to mirror

Reply:
😅